src/spikeinterface/extractors/nwbextractors.py

```diff
@@ -606,6 +624,12 @@ def __init__(
             # not json serializable if file arg is provided
             self._serializability["json"] = False
 
+        if nwbfile is not None:
+            # NWBFile is not serializable
+            self._serializability["memory"] = False
+            self._serializability["json"] = False
+            self._serializability["pickle"] = False
```

Comment:
@CodyCBakerPhD note that this will make the object unsuitable for parallel processing (so this approach is not recommended IMO)